test/core/Leaf.t.sol

```diff
@@ -351,7 +357,7 @@ contract FieldTest is Test {
     for (; i < 256; i++) {
       if (uint(b >> i) % 2 == 1) break;
     }
-    assertFirstOnePosition(uint(b), i);
+    assertFirstOnePosition(uint(b), i > MAX_LEVEL_SIZE ? MAX_LEVEL_SIZE : i);
```

Comment:
I found this surprising. Would it perhaps be more natural to split this into two assertions? Or am I missing something?

```suggestion
    assertTrue(i < MAX_LEVEL_SIZE);
    assertFirstOnePosition(uint(b), i);
```

Reply:
So before `ctz` would return the number of trailing zeroes for any word. Now it maxes out at 64. But the loop just before the assertions may push `i` to 255, so `assertTrue(i < MAX_LEVEL_SIZE);` would fail. Would this be OK?

```
if (i > MAX_LEVEL_SIZE) {
  assertFirstOnePosition(uint(b),MAX_LEVEL_SIZE);
} else {
assertFirstOnePosition(uint(b), i);
}
```

Reply:
Ah, now I get it. I think it's fine as is 👍